bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/Marker.java

```diff
@@ -95,7 +105,7 @@ public Severity getSeverity() {
         return severity;
     }
 
-    public Option<Union<Interval>> getUnion() { return union; }
+    public Option<Pair<ImList<String>, Union<Interval>>> getUnion() { return union; }
```

Comment:
I doubt this is used anywhere?  It isn't really just a `Union` anymore regardless.  We might just get rid of it.